doc/try-kadalu.md

```diff
@@ -129,4 +129,4 @@ As long as glusterfs promises to keep the backend layout same, and continue to p
 
 ### Gluster and Kadalu
 
-We have compiled a list of things [here](./gluster-and-kadalu.md)
+We have compiled a list of things [here](https://kadalu.io/docs/k8s-storage/master/gluster-and-kadalu)
```

Comment:
this may break the version specific documentation! Give me some time to review how to get this..

Reply:
I thought that this particular segment is unlikely to be versioned and is complete at this point.

Reply:
@amarts removing `.md` works in the generated site. Adding below code after https://github.com/kadalu/docs/blob/master/generate.sh#L30 will fix the issue.

```
sed -i -e "s#.md)#)#g" doc/*.md
```

Any suggestions to make above code safe to avoid false substitutions?